schwifty/bank_registry/manual_it.json

```diff
@@ -1,666 +1,1122 @@
+	},
+	{
+		"bank_code": "05142",
+		"bic": "BCPTITNN",
+		"country_code": "IT",
+		"name": "BANCA DI CREDITO POPOLARE",
+		"primary": true,
+		"short_name": "BANCA DI CREDITO POPOLARE"
+	},
+	{
+		"bank_code": "05156",
+		"bic": "BCPCIT2P",
+		"country_code": "IT",
+		"name": "BANCA DI PIACENZA - SOCIETA' COOPERATIVA PER AZIONI",
+		"primary": true,
+		"short_name": "BANCA DI PIACENZA"
+	},
+	{
+		"bank_code": "05262",
+		"bic": "BPPUIT33",
+		"country_code": "IT",
+		"name": "BANCA POPOLARE PUGLIESE - SOCIETA' COOPERATIVA PER AZIONI",
+		"primary": true,
+		"short_name": "BANCA POPOLARE PUGLIESE"
+	},
+	{
+		"bank_code": "05297",
+		"bic": "BPFRIT3F",
+		"country_code": "IT",
+		"name": "Banca Popolare del Frusinate",
+		"primary": true,
+		"short_name": "Banca Popolare del Frusinate"
+	},
+	{
+		"bank_code": "05372",
+		"bic": "POCAIT3C",
+		"country_code": "IT",
+		"name": "Banca Popolare del Cassinate S.c.p.A.",
+		"primary": true,
+		"short_name": "Banca Popolare del Cassinate"
+	},
+	{
+		"bank_code": "05385",
+		"bic": "BPDMIT3B",
+		"country_code": "IT",
+		"name": "BANCA POPOLARE DI PUGLIA E BASILICATA - SOCIETA' COOPERATIVA PER AZIONI",
+		"primary": true,
+		"short_name": "BANCA POPOLARE DI PUGLIA E BASILICATA"
+	},
+	{
+		"bank_code": "05387",
+		"bic": "BPMOIT22",
+		"country_code": "IT",
+		"name": "BPER Banca S.p.A",
+		"primary": true,
+		"short_name": "BPER Banca"
+	},
+	{
+		"bank_code": "05424",
+		"bic": "BPBAIT3B",
+		"country_code": "IT",
+		"name": "BANCA POPOLARE DI BARI - SOCIETA' PER AZIONI",
+		"primary": true,
+		"short_name": "BANCA POPOLARE DI BARI"
+	},
+	{
+		"bank_code": "05428",
+		"bic": "BLOPIT22",
+		"country_code": "IT",
+		"name": "BANCA POPOLARE DI BERGAMO S.P.A.",
+		"primary": true,
+		"short_name": "BANCA POPOLARE DI BERGAMO"
+	},
+	{
+		"bank_code": "05484",
+		"bic": "CIVIIT2C",
+		"country_code": "IT",
+		"name": "BANCA DI CIVIDALE S.P.A",
+		"primary": true,
+		"short_name": "BANCA DI CIVIDALE"
+	},
+	{
+		"bank_code": "05652",
+		"bic": "SFSPIT22",
+		"country_code": "IT",
+		"name": "SANFELICE 1893 BANCA POPOLARE",
+		"primary": true,
+		"short_name": "SANFELICE 1893 BANCA POPOLARE"
+	},
+	{
+		"bank_code": "05676",
+		"bic": "BPSAIT31",
+		"country_code": "IT",
+		"name": "BIBANCA S.P.A.",
+		"primary": true,
+		"short_name": "BIBANCA"
+	},
+	{
+		"bank_code": "05696",
+		"bic": "POSOIT22",
+		"country_code": "IT",
+		"name": "BANCA POPOLARE DI SONDRIO",
+		"primary": true,
+		"short_name": "BANCA POPOLARE DI SONDRIO"
+	},
+	{
+		"bank_code": "05772",
+		"bic": "PSANIT3P",
+		"country_code": "IT",
+		"name": "BANCA POPOLARE SANT'ANGELO SCPA",
+		"primary": true,
+		"short_name": "BANCA POPOLARE SANT'ANGELO"
+	},
+	{
+		"bank_code": "05856",
+		"bic": "BPAAIT2B",
+		"country_code": "IT",
+		"name": "BANCA POPOLARE DELL'ALTO ADIGE SOCIETA' PER AZIONI (IN TEDESCO SUDTIROLER VOLKSBANK A.G.)",
+		"primary": true,
+		"short_name": "BANCA POPOLARE DELL'ALTO ADIGE SOCIETA"
```

Comment:
[low]: we can remove `SOCIETA` as well since it's part of the `S.P.A.` of the full name